code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -1055,6 +1083,7 @@
 	reagent_state = SOLID
 	color = "#555555"
 	overdose_threshold = 30
+	process_flags = ORGANIC | SYNTHETIC
```

Comment:
It's important to make sure you have covered any reagents we ourselves have added. Have you done this?

Reply:
i checked the entire toxins/medicine/other reagent code, i added flags for the ones i think should affect them of these new ones, i havent checked the alcohol dm though.

Reply:
if anything, i was thinking of adding that they can process neurine, or a synth counterpart, because otherwise they are stuck on hallucinations forever if the SM blows up.